specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimnotifications.json

```diff
@@ -432,6 +435,9 @@
         "responses": {
           "204": {
             "description": "The Users is subscribed to receive the notification."
+          },
+		  "404": {
+            "description": "The Users is not subscribed to receive the notification."
```

Comment:
>description [](start = 13, length = 11)

dont add this. If you want the description, add it to the default description below